gensim/__main__.py

```diff
@@ -0,0 +1,19 @@
+import platform
+import sys
+import numpy
+import scipy
+import gensim
+import argparse
+
+if __name__ == "__main__":
+
+        parser = argparse.ArgumentParser()
+        parser.add_argument("--info", action="store_true", help="Display version of Gensim and its dpendencies")
+        opt = parser.parse_args()
+        if opt.info:
+                print(platform.platform())
```

Comment:
Please move it do distinct function (that returns `dict` with information)
```python

from gesnim import package_info

info = package_info()
```

and print it here in nice manner